### ISSUE TITLE
Create generic python script to test a model

### ISSUE THREAD

**Issue by thehoul** [kelansolo/Large-Rocks-Detection#7]
Create a Python script that should take as input a model (yolov8n.pt, yolo11n.pt, etc...) and a data file (e.g. `Pierre.yaml`) and a number of epochs and automatically creates a model from the pre-trained model, train it on the data file, then validate it and run tests and return some metrics (e.g. the accuracy). The output should probably be saved in a file that is also given as input. 

The goal is to be able to use this script to run batches on the SCITAS machines (see README) to quickly be able to assess the quality of a model and/or a dataset. For instance, if we want to test later what merging of the data creates the best output, we should be able to only modify the dataset and then using this script run a batch and see how well it performed.
